azuredevops/internal/service/serviceendpoint/data_serviceendpoint_azurerm.go

```diff
@@ -23,7 +23,7 @@ func dataSourceServiceEndpointAzureRMRead(d *schema.ResourceData, m interface{})
 	if serviceEndpoint != nil {
 		(*serviceEndpoint.Data)["creationMode"] = ""
 		d.Set("service_endpoint_id", serviceEndpoint.Id.String())
-		flattenServiceEndpointAzureRM(d, serviceEndpoint, projectID)
+		flattenServiceEndpointAzureRM(d, serviceEndpoint, projectID, false)
```

Comment:
The `validation` flag is a feature of the resource,   data source can ignore it and the data source schema not defined this property. There is also no `validation` in the API response, `true` or `false` does no matter here.